ReleaseNotes_V1.1.txt

```diff
@@ -0,0 +1,175 @@
+
+Change Description
+==================
+
+  This new version (1.1) of the ACIS Focal Plane Thermal Model has
+  three categories of changes:
+
+    I) Create the run.dat file regardless of the value of the verbose
+       switch
+ 
+   II) Do not label the run "NOT OK" when a preference for a cold and
+       stable focal plane temperature has not been met. Also, whether
+       the preference was met or not met, the report used the term
+       "violation" This was changed to "Preference Not Met".
+
+  III) Split out ACIS-I observations from ACIS-S and assign different
+       violation temperatures to the two different categories.
+
+Rationale
+==========
+    I) It was observed that the model did not write out the run.dat file
+     for cetain values of the verbose switch.  We want the run.dat
+     file to be created for any verbosity level.
+
+   II) ACIS observations with certain characteristics have a preference
+      for a cold and stable focal plane temperature. Prior to this
+      update,  when the preference was not met the Status output by
+      the model was "NOT OK".  As this is only a preference and not a 
+      requiremnent of the observation, the output status should be
+      "OK" even if the preference is not met. Likewise, use of the
+      term "violation" was too strong. 
+
+
+  III) After much discussion, the idea of changing the violation
+       temperature of all ACIS-S observations to -112.0 deg. C was approved.
+       The violation temperature for ACIS-I observations would remain
+       unchanged at -114.0 deg. C.  Now that this change is
+       operational, the behavior of the ACIS Focal Plane Predictive
```

Comment:
The model reporting is affected. Previously, there was a statement in 
the HTML file if -114 was violated, and the FP_TEMP status line at the 
top would say NOT OK.

Now  there are separate messages for ACIS-I (-114) and ACIS S (-112)  
limits was violated. If either (or both) of those limits were violated 
by a science observation, the line would read differently and there 
would be the usual table underneath the line (like you see for -118.7 
perigee passages).

Attached is a screen shot of the new HTML file, for when the status is ok


Gregg


On 05/18/2016 10:47 AM, Tom Aldcroft wrote:
>
> In ReleaseNotes_V1.1.txt 
> <https://github.com/acisops/acisfp_check/pull/3#discussion_r63716898>:
>
> > +     file to be created for any verbosity level.
> > +
> > +   II) ACIS observations with certain characteristics have a preference
> > +      for a cold and stable focal plane temperature. Prior to this
> > +      update,  when the preference was not met the Status output by
> > +      the model was "NOT OK".  As this is only a preference and not a
> > +      requiremnent of the observation, the output status should be
> > +      "OK" even if the preference is not met. Likewise, use of the
> > +      term "violation" was too strong.
> > +
> > +
> > +  III) After much discussion, the idea of changing the violation
> > +       temperature of all ACIS-S observations to -112.0 deg. C was approved.
> > +       The violation temperature for ACIS-I observations would remain
> > +       unchanged at -114.0 deg. C.  Now that this change is
> > +       operational, the behavior of the ACIS Focal Plane Predictive
>
> Do you mean the load checker tool, not the predictive model? The model 
> itself doesn't care about limits.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/acisops/acisfp_check/pull/3/files/09044a9671e523b7b0d3bc836b549d268850beca#r63716898>
>